_std/defines/ocean_generation.dm

```diff
@@ -0,0 +1,3 @@
+// #define OCEAN_COLOR "#4DA0FD"
```

Comment:
at least put this in `watermap.dm` or something

Reply:
is that an existing file? or should i rename it

Reply:
renamed